quickwit/quickwit-indexing/src/source/kafka_source.rs

```diff
@@ -667,12 +667,17 @@ fn create_consumer(
     params: KafkaSourceParams,
     events_tx: mpsc::Sender<KafkaEvent>,
 ) -> anyhow::Result<(ClientConfig, RdKafkaConsumer)> {
-    let mut client_config = parse_client_params(params.client_params)?;
-
     // Group ID is limited to 255 characters.
-    let mut group_id = format!("quickwit-{index_uid}-{source_id}");
+    let mut group_id = params
```

Comment:
Actually, this might be easier to read:
```rust
let mut group_id = match &params.client_params["group.id"] {
        JsonValue::String(group_id) => group_id.clone(),
        _ => format!("quickwit-{index_uid}-{source_id}"),
    };
```